docker-compose.yml

```diff
@@ -12,4 +12,17 @@ services:
     # Run a command against the development stage of the image
     command: npm run start:dev
     ports:
-      - 3000:3000
\ No newline at end of file
+      - 3000:3000
+
+  db:
+    image: mysql
+    restart: always
+    container_name: nest-mysql
+    environment:
+      MYSQL_DATABASE: nest
+      MYSQL_USER: root
+      MYSQL_PASSWORD: ${MYSQL_PASSWORD}
```

Comment:
e xem dùng chung file env vs application nhé
https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option
thì đỡ phải định nghĩa lại như này